mkosi/config.py

```diff
@@ -3345,6 +3346,14 @@ def parse_kernel_module_filter_regexp(p: str) -> str:
         default=True,
         help="Automatically enroll the secureboot signing key on first boot",
     ),
+    ConfigSetting(
+        dest="secure_boot_auto_enroll_certificates",
+        long="--secure-boot-auto-enroll-certificates",
+        metavar="PATH",
+        section="Validation",
+        parse=config_make_list_parser(delimiter=",", parse=make_path_parser()),
+        help="List of comma-separated paths to PEM certificates to be automatically enrolled in UEFI db",
```

Comment:
```suggestion
        help=,argparse.SUPPRESS
```
so it's not show in the help.